### ISSUE TITLE
TypeError: undefined is not a function

### ISSUE THREAD

**Issue by nahtnam** [AtomLinter/linter-eslint#230]
Hey!

I am getting this issue:
```
TypeError: undefined is not a function
  at Object.provider.lint (/home/nahtnam/.atom/packages/linter-eslint/lib/linter-eslint.coffee:83:24)
  at Promise.then._this.emitter.emit.linter (/home/nahtnam/.atom/packages/linter/lib/linter-registry.coffee:59:22)
  at LinterRegistry.triggerLinter (/home/nahtnam/.atom/packages/linter/lib/linter-registry.coffee:58:16)
  at /home/nahtnam/.atom/packages/linter/lib/linter-registry.coffee:52:17
  at Array.map (native)
  at /home/nahtnam/.atom/packages/linter/lib/linter-registry.coffee:50:27
```

Package.json
```
{
  "name": "counterstrike",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "start": "node ./bin/www",
    "dev": "nodemon ./bin/www"
  },
  "dependencies": {
    "body-parser": "~1.13.2",
    "bookshelf": "^0.8.2",
    "cookie-parser": "~1.3.5",
    "cors": "^2.7.1",
    "debug": "~2.2.0",
    "express": "~4.13.1",
    "knex": "^0.8.6",
    "moment": "^2.10.6",
    "morgan": "~1.6.1",
    "node-schedule": "^0.2.9",
    "pg": "^4.4.2",
    "request": "^2.65.0"
  },
  "devDependencies": {
    "eslint": "^1.6.0",
    "nodemon": "^1.7.1"
  }
}

```

eslintrc
```
{
    "rules": {
        "indent": [
            2,
            2,
            {"SwitchCase": 1}
        ],
        "quotes": [
            2,
            "single"
        ],
        "linebreak-style": [
            2,
            "unix"
        ],
        "semi": [
            2,
            "always"
        ],
        "no-redeclare": [
          0
        ]
    },
    "env": {
        "es6": true,
        "node": true
    },
    "extends": "eslint:recommended"
}

```

Thanks.

**Comment by nahtnam** [comment]
Atom Version: 1.0.19
Running the latest version of this addon.
Node: v4.1.1
NPM: 2.14.5

**Comment by w-green** [comment]
I'm seeing the same error with Atom version 1.1.0

**Comment by steelbrain** [comment]
Fixed by v5.0.0
